pages/documentation/developer/repositories/powsybl-diagram.md

```diff
@@ -1,25 +1,53 @@
-# powsybl-single-line-diagram [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-single-line-diagram.svg?sort=semver)](https://github.com/powsybl/powsybl-single-line-diagram/releases/)
-This [repository](https://github.com/powsybl/powsybl-single-line-diagram) provides modules to generate single line diagrams.
+# powsybl-diagram [![GitHub release](https://img.shields.io/github/release/powsybl/powsybl-diagram.svg?sort=semver)](https://github.com/powsybl/powsybl-diagram/releases/)
+The powsybl-diagram [repository](https://github.com/powsybl/powsybl-diagram) provides features to generate single-line diagrams and network-area diagrams.
+
+## Features
+
+### powsybl-single-line-diagram
+This [artifact](https://github.com/powsybl/powsybl-diagram/powsybl-single-line-diagram) provides features to generate customized single-line diagrams:
```

Comment:
the link is not working, you want to link to existing code? then it's https://github.com/powsybl/powsybl-diagram/tree/main/network-area-diagram. But I think you can remove the link.